### DISCUSSION TITLE
IMU reference orientation

### DISCUSSION THREAD

**Discussion by Robatic2021** [FieldRobotEvent/Virtual_Field_Robot_Event#12] [General]
Hello all,

We, the Robatic team, observed that the IMU in the simulation environment has an absolute reference orientation. The yaw of the Jackal remains constant, despite starting it up under different yaw angles. To us, this seemed a bit unrealistic, as an IMU should create its reference orientation when started up, meaning that it will reference all its movements to the orientation at start-up. Is this thought correct, and if so, are there plans to change the current reference orientation?

It would make a difference in task 1 and 2, in which the starting orientation is undefined. Using the absolute, independent reference orientation of the simulation environment then feels a bit like cheating ;-).

We hope someone can help us,
Best regards,

The Robatic team 2021

**Comment by ThijsRuigrok** [comment]
Dear Robatic2021,

Thank you for reporting this issue. We acknowledge this issue. We currently do not have a proper solution available. We recommend you to not rely on the absolute reference of the IMU as we might come up with a solution later. 

Kind regards,
Thijs
